services/command-router/pom.xml

```diff
@@ -73,6 +73,10 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
           <groupId>io.jaegertracing</groupId>
           <artifactId>jaeger-client</artifactId>
         </dependency>
+        <dependency>
+          <groupId>io.jaegertracing</groupId>
+          <artifactId>jaeger-micrometer</artifactId>
+        </dependency>
```

Comment:
IMHO we also need to add the dependency to the Quarkus based version ...

Reply:
It looks to me as if jaeger client metrics are already supported via the `quarkus-jaeger` dependency and can be enabled by setting `quarkus.jaeger.metrics.enabled` to `true`.

I think this property should be added in the default quarkus config in the IoT Packages chart. I'll create a separate PR there then.

Reply:
My understanding is that the Quarkus Jaeger extension does not per se pull in the dependency but can make use of it, if the Quarkus Metrics extension is being used. Since we do not use that extension, I do not think that the jar is being pulled in as a quick run of the Maven Dependency plugin in for `command-router-quarkus` seems to confirm:
```bash
mvn dependency:list -Pbuild-docker-image,metrics-prometheus,jaeger | grep jaeger
[INFO]    io.quarkus:quarkus-jaeger:jar:1.11.3.Final:compile -- module quarkus.jaeger (auto)
[INFO]    io.jaegertracing:jaeger-core:jar:1.5.0:runtime -- module jaeger.core (auto)
[INFO]    io.jaegertracing:jaeger-thrift:jar:1.5.0:runtime -- module jaeger.thrift (auto)
```
and
```bash
mvn dependency:list -Pbuild-native-image,metrics-prometheus,jaeger | grep jaeger
[INFO]    io.quarkus:quarkus-jaeger:jar:1.11.3.Final:compile -- module quarkus.jaeger (auto)
[INFO]    io.jaegertracing:jaeger-core:jar:1.5.0:runtime -- module jaeger.core (auto)
[INFO]    io.jaegertracing:jaeger-thrift:jar:1.5.0:runtime -- module jaeger.thrift (auto)
```

Reply:
The `jaeger-micrometer` dependency only contains one class - `MicrometerMetricsFactory`.
In the quarkus jaeger code, I see a corresponding `QuarkusJaegerMicrometerFactory` class in the quarkus-jaeger module itself getting used instead (see [here](https://github.com/quarkusio/quarkus/blob/59b7449ffb90304ef7c17cf9175ea2d10b74b80a/extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/QuarkusJaegerMicrometerFactory.java#L16), instantiated [here](https://github.com/quarkusio/quarkus/blob/b96973dcbcbff76ec6842448ba802e1269b5ca53/extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentRecorder.java#L39)). So I don't think the `jaeger-micrometer` dependency is needed for quarkus.